src/devices/gridstream.c

```diff
@@ -0,0 +1,258 @@
+/** @file
+    Decoder for Gridstream RF devices produced by Landis & Gyr.
+
+    Copyright (C) 2016 Benjamin Larsson
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+ */
+
+/**
+Landis & Gyr Gridstream Power Meters.
+
+- Center Frequency: 915 Mhz
+- Modulation: FSK-PCM
+- Bitrates: 9600, 19200, 38400
+- Preamble: 0xAAAA
+- Syncword v4: 0b0000000001 0b0111111111
+- Syncword v5: 0b0000000001 0b11111111111
+
+Datastream is variable length and bitrate depending on type fields
+Preamble
+Bytes after preamble are encoded with 8N1
```

Comment:
Elaborate that this is standard uart settings and that the actual bit stream is start bit, data and stop bit.

Reply:
Add a link to the protocol wiki: https://wiki.recessim.com/view/Landis%2BGyr_GridStream_Protocol and a note that the decoder is based on information from there.

Reply:
Addressed in https://github.com/merbanan/rtl_433/pull/2616/commits/7edd93a5aead3c669f4bd8e5dbf2d6d32e9ede96